CONTRIBUTING.md

```diff
@@ -140,56 +220,7 @@ In case of conflicts between your branch and `dev` you must solve them either on
 4. 📝 The conflicts have to be [manually](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line) resolved
     
 
-### Conventions for git and GitHub
-
-🔶 Naming convention for branches
-
-Follow the [best-practices for git branching naming convention](https://codingsight.com/git-branching-naming-convention-best-practices/).
-The convention is `type`-`issue-nr`-`short-description`
-
-**type**
-
-* master / main / stable - includes the current stable version
-* dev - includes all current developments
-* feature - includes the feature that will be implemented
-* release - includes the current version to be released
-
-The majority of the ontology development will be done in `feature` branches.
-
-**issue-nr**
-
-Add the issue number where you describe, discuss and document your development.
-
-**short-description**
-
-Describe shortly what the branch is about. Avoid long and short descriptive names for branches, 2-4 words are optimal.
-
-Other hints:
-- Separate words with `-` (minus)
-- Do not put your name to the branch name, it's a collaborative project
-- Avoid using numbers only 
-- Branch names should be precise and informative
-- Avoid using capital letters
-
-Examples of branch names : `feature-43-add-new-ontology-class` or `feature-711-branch-naming-convention`
-
-🔶 Commit messages
-
-Try to follow esxisting conventions for commit messages:
-- [How to Write a Git Commit Message](https://chris.beams.io/posts/git-commit/)
-- Keep the subject line [short](https://chris.beams.io/posts/git-commit/#limit-50) (i.e. do not commit more than a few changes at the time)
-- Use [imperative](https://chris.beams.io/posts/git-commit/#imperative) for commit messages 
-- Do not end the commit message with a [period](https://chris.beams.io/posts/git-commit/#end) 
-
-
-You can use the command 
-```bash
-git commit --amend
-```
-to edit the commit message of your latest commit (provided it is not already pushed on the remote server). <br>
-With `--amend` you can even add/modify changes to the commit.
-
 
 ### Communicate
 
-Feel free to ask the community if you need help. We are happy to support you.
+Feel free to ask the community if you need help. We are happy to support you. If you want to contact specific teammembers or experts, please check out the Team section of our [README](https://github.com/OpenEnergyPlatform/ontology/blob/dev/README.md) file. Use the [OEP contact form](https://openenergyplatform.org/contact/) if you want to become a team member.
```

Comment:
Might be nice to directly link to said section? Also, strictly speaking it is the `Teams` section.
```suggestion
Feel free to ask the community if you need help. We are happy to support you. If you want to contact specific teammembers or experts, please check out the Teams section of our [README](https://github.com/OpenEnergyPlatform/ontology?tab=readme-ov-file#teams) file. Use the [OEP contact form](https://openenergyplatform.org/contact/) if you want to become a team member.
```

Reply:
Although the teams need a rework aswell ...